src/app/[lang]/suggestions/page.tsx

```diff
@@ -1,15 +1,61 @@
+'use client';
 import { PostPage, PostsContainer, PostsForm } from '@shared';
+import { useState } from 'react';
+import {suggestionPost} from '@/types';
 
 const Page = () => {
+    const [posts, setPosts] = useState<suggestionPost[]>([]);
+    const [post, setPost] = useState<suggestionPost>({ title: '', content: '', category: '' });
+    const addPost = (e: React.MouseEvent<HTMLButtonElement>): void => {
+      e.preventDefault();
```

Comment:
``` e.preventDefault(); ```

is useless here since the event is called on button.onClick which does nothing in default, therefore e param can be removed